verification/curator-service/ui/src/redux/@types/react-redux.d.ts

```diff
@@ -0,0 +1,7 @@
+import 'react-redux';
```

Comment:
What was the purpose of adding those custom types?

Reply:
I was getting ts errors, and I found that it is a good practice to add that custom type